portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+/** 
+ * Toggles the drop down text.
+ */
+function toggleDropDown(id) {
+  document.getElementById(id).classList.toggle("show");
+}
+
+/**
+ * Adds a random favorite quote to the page.
+ */
+function addRandomFavoriteQuote() {
+  const favoriteQuotes =
+      ["Many years later, as he faced the firing squad, Colonel Aureliano Buendía was to remember that distant afternoon when his father took him to discover ice. \n —Gabriel García Márquez, One Hundred Years of Solitude",
```

Comment:
We use single quotes in JavaScipt instead of double.
Here and everywhere else. 

Refer: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
Done